server/src/test/java/io/crate/expression/ValueExtractorsTest.java

```diff
@@ -1,50 +0,0 @@
-/*
- * Licensed to Crate.io GmbH ("Crate") under one or more contributor
- * license agreements.  See the NOTICE file distributed with this work for
- * additional information regarding copyright ownership.  Crate licenses
- * this file to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.  You may
- * obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial agreement.
- */
-
-package io.crate.expression;
-
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-
-import java.util.Collections;
-import java.util.Map;
-import java.util.function.Function;
-
-import org.junit.Test;
-
-import io.crate.metadata.ColumnIdent;
-import io.crate.types.DataTypes;
-
-public class ValueExtractorsTest {
-
-    @Test
-    public void testIntegerAreConvertedToMatchType() {
```

Comment:
Is this test scenario already covered by e.g. sourcelookup tests?

Reply:
Actually this test calls a method which is used only in tests. And basically it doesn't really test `ValueExtractors`, it's rather tests that `type.implicitCast` works as expected which is irrelevant for value extraction. 

I don't think we need this test